src/gmail.js

```diff
@@ -578,24 +578,23 @@ var Gmail = function(localJQuery) {
 
 
     api.get.unread_inbox_emails = function() {
-        var dom = $("div[role=navigation]").find("[title*='" + api.tools.i18n("inbox") + "']");
-
-        if(dom.length > 0) {
-            if(dom[0].title.indexOf("(") !== -1) {
-                return parseInt(dom[0].title.split(":")[0].replace(/[^0-9]/g, ""));
+            var dom = $("div[role=navigation]").find("[title*='" + api.tools.i18n("inbox") + "']");
+            if(dom.length > 0) {
+                if(dom[0].title.indexOf("Inbox") !== -1) {
+                    return parseInt(dom[0].attributes['aria-label'].value.replace(/[^0-9]/g, ""));
+                }
             }
-        }
-
-        return 0;
-    };
-
+    
+            return 0;
+        };
+    
 
     api.get.unread_draft_emails = function() {
         var dom = $("div[role=navigation]").find("[title*='" + api.tools.i18n("drafts") + "']");
 
         if(dom.length > 0) {
-            if(dom[0].title.indexOf("(") !== -1) {
-                return parseInt(dom[0].title.replace(/[^0-9]/g, ""));
+            if(dom[0].title.indexOf("Drafts") !== -1) {
+                return parseInt(dom[0].attributes['aria-label'].value.replace(/[^0-9]/g, ""));
```

Comment:
This will only work with English language. Maybe api.tools.i18n() can be used here?